.travis.yml

```diff
@@ -27,4 +18,4 @@ sudo: false
 script:
   - mix test
   - mix credo
-  - if [[ `elixir -v` = *"1.6"* ]]; then mix format --check-formatted; fi
+  - if [[ `elixir -v` = *"1.7"* ]]; then mix format --check-formatted; fi
```

Comment:
Why did you change this to 1.7?

Reply:
1.7, being the latest public release (outside of 1.8-rc1), felt like the logical choice to run this particular build step on — especially since I'm adding it to the build.

Reply:
This is meant to ensure the command isn't run without formatter (versions prior to 1.5).